shared/teams/team/rows/emoji-row/add.tsx

```diff
@@ -25,7 +25,17 @@ const AddEmoji = ({teamID, convID, filter, setFilter}: OwnProps) => {
         ],
       })
     )
-  const onAddAlias = () => {}
+  const onAddAlias = () =>
```

Comment:
would be good to hook this up to the "add alias"/"edit alias" menu buttons in `emoji-row/item.tsx`